src/routes/mod.rs

```diff
@@ -229,7 +231,8 @@ pub enum InvocationResponse {
 impl<'r> Responder<'r, 'static> for InvocationResponse {
     fn respond_to(self, request: &'r Request<'_>) -> rocket::response::Result<'static> {
         match self {
-            InvocationResponse::Empty => ().respond_to(request),
+            InvocationResponse::NotFound => Option::<()>::None.respond_to(request),
+            InvocationResponse::EmptySuccess => ().respond_to(request),
```

Comment:
NABD but this could be clearer using the impl for `(Status, R)`.
```suggestion
            InvocationResponse::NotFound => (Status::NotFound, ()).respond_to(request),
            InvocationResponse::EmptySuccess => ().respond_to(request),
```

Reply:
Merged before I realised there was another comment :(